src/client/components/pages/Courses/CoursesPage.tsx

```diff
@@ -108,24 +134,42 @@ const CoursesPage: FunctionComponent = (): ReactElement => {
           </div>
         )
         : (
-          <CourseInstanceTable
-            academicYear={acadYear}
-            courseList={
-              showRetired
-                ? currentCourses
-                : currentCourses.filter(
-                  ({ spring, fall }): boolean => (
-                    fall.offered !== OFFERED.RETIRED
-                        && spring.offered !== OFFERED.RETIRED)
+          <>
+            <Dropdown
+              id="select-view-dropdown"
+              name="select-view-dropdown"
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setCurrentViewId(event.currentTarget.value);
+              }}
+              labelPosition={POSITION.LEFT}
+              label="View"
+              value={currentViewId}
+              options={[
+                ...views.map((view) => ({
+                  label: view.name,
+                  value: view.id,
+                })),
+              ]}
```

Comment:
The spread operator isn't necessary here since `Array.map` returns a new array. It could just be:

```tsx
  options={views.map((view) => ({
    label: view.name,
    value: view.id,
  })
}
```

Reply:
Good point. Fixed that in f20c13236dbc628f885adef79e103845a72f8401